\label{chap:genetic}
\starredchapter{APPENDIX~\thechapter. GENETIC STOCK STRUCTURE}
\lfoot{\fishName}
\rfoot{Appendix~\thechapter~-- Genetic stock structure}

\appsection{USE OF GENETIC STOCK ID FOR MANAGEMENT}

One of the main goals of fisheries genetics is to determine the boundaries of putative fish stocks that can in turn be used to determine management units (Hauser and Carvalho 2008).
Genetic techniques can be used to assess the rate of gene flow or migration between regions which can in turn provide insight into the connectivity between populations and aid in determining population boundaries.
Determining population boundaries is important for matching biological populations with management units (Reiss et al. 2009).
Here we use the term stock to refer to a management unit and population to refer to a group that is defined by significant and reproducible genetic differentiation, which is largely demographically independent from other such groups (after Reiss et al. 2009).
Understanding gene flow can also provide insight into source-sink dynamics, including the relationship between larval dispersal and adult populations (Hauser and Carvalho 2008).
Maintenance of multiple populations, which may be adapted to local conditions, may be important for the long term resilience of a species and may contribute to the overall productivity of a fishery (Hilborn et al. 2003).

Identifying genetic structure in pelagic fish species can be challenging because such species tend to have large population census sizes with few (if any) barriers to dispersal as either juveniles or adults (Ward et al. 1994; Waples 1998).
Thus, gene flow is often relatively unrestricted and, even if barriers to gene flow do exist, large populations tend to maintain genetic diversity which makes detecting restrictions difficult.
However, an increasing number of studies have detected population structure in pelagic fishes, with evidence that this structure is biologically relevant, i.e., structure corresponds to a change in environmental parameters or to differences in a physiological trait (Atlantic Herring, Bekkevold et al. 2005; Atlantic Cod, Knutsen et al. 2011; Pacific Herring, Wildes et al. 2011).

Molecular markers offer a powerful tool for estimating gene flow.
A variety of markers have been used in marine fishes including allozymes, mitochondrial DNA (mtDNA), microsatellites and, most recently, single nucleotide polymorphisms (SNPs).
See Glossary for definitions.
While all have their strengths and weaknesses, microsatellites are often favored for use over other markers due to their high degree of variability and ease of genotyping in the laboratory.
In comparisons with allozymes, it has been demonstrated that microsatellites have a smaller sampling variance and greater statistical power for detecting population structure (Waples 1998; Larsson et al. 2007).

Several statistics can be used to assess population structure from genetic data.
The most common is pairwise FST.
FST is a measure of the amount of inbreeding that occurs within a group or population compared to other groups or populations (see longer description in Glossary).
FST values range from 0 (complete panmixia) to 1 (total contemporary and historical isolation).
Because the amount of inbreeding is affected by population size (i.e., inbreeding is greater in small populations), pelagic marine fishes, which typically have extremely large population sizes on the orders of millions of individuals, tend to have very low FST values.
Ward et al. (1994) found that the median FST of marine fishes, calculated from a variety of species, was 0.02 and significant FST values less than 0.01 have been observed in some species (e.g., Bekkevold et al. 2005; White et al. 2010).
FST is calculated based on the frequency of shared alleles between groups and significant pairwise FST values indicate whether the level of population differentiation between the groups under consideration is statistically greater than zero.
Significance is typically determined by permutation testing (Welch 1990), with allele frequencies pooled and resampled in order to recalculate the test statistic (in this case FST).
It must be noted that statistical significance does not necessarily mean there is an important biological difference between groups.
Statistical, but not biological, significance could arise due to disproportionately sampling within a family (i.e., non-random sampling) or by detection of differences that are not large enough to be relevant to management, often by using a large number of loci (Waples 1998).
The converse problem may also occur (Waples 1998; Hauser and Carvalho 2008).
For example, in large marine populations, low FST values may correspond to relatively low migration rates on the order of a few 100 individuals, which may be <1\% of the population.
However, restricted migration, with rates of between 0.2 and 10\%, has been shown to be sufficient for differential response to demographic perturbation, yet may be too low to result in significant genetic structure (Hauser and Carvalho 2008, and references therein).
In addition, neutral markers reflect changes over evolutionary time and may not identify recent isolation of populations (Hauser and Carvalho 2008).
Despite these caveats, FST is one of the most commonly used methods for quantifying genetic difference between groups and has been used to address questions relevant to the management of marine species (e.g., White et al. 2009; Selkoe et al. 2010; Bekkevold et al. 2011).

\appsection{LIMITED DISPERSAL AND POPULATION STRUCTURE}

The range of \fishName{} extends along coastal shelf regions from Washington State along the northwestern coast of North America to the Bering Sea and along the north eastern coast of Asia.
\fishName{} form concentrations in defined regions and follow predictable patterns of yearly migration (Ketchen 1961; Shimada and Kimura 1994).
The results of several tagging studies suggest that \fishName{} have a limited dispersal range, although migrations of long distances have been observed (Shimada and Kimura 1994; Gustafson et al. 2000).
Thus, connectivity between regions is likely limited, which may result in stock structuring.

\appsection{RESEARCH QUESTION}

Currently, \fishName{} are managed as four stocks - West Coast Vancouver Island (WCVI), Strait of Georgia (SoG), Queen Charlotte Sound (QCS) and Hecate Strait (HS) (Sinclair and Starr 2005).
Although previous genetic work has identified a difference between the fish from the Strait of Georgia and those from the Gulf of Alaska and the Bering Sea, it is unknown whether the Canadian populations are genetically distinct.
And if, in particular, the Hecate Strait and Queen Charlotte Sound stocks are part of larger populations extending south from Alaska.
The previous \fishName{} assessment specifically recommended more detailed analysis of the Queen Charlotte Sound stock as there was little evidence at the time to warrant the designation of this stock as unique from other populations (Sinclair and Starr 2005).
Here, we provide a review of the current state of understanding of the genetic structure of \fishName{} and describe future research which could address the following two questions:
(1) are populations from WCVI, SoG, QCS, and HS genetically distinct from each other and from US stocks, and
(2) does fishing occur on mixed stocks in QCS and/or HS?

\appsection{SUMMARY OF PREVIOUS GENETIC STUDIES}

In the earliest assessment of genetic population structure in \fishName{}, allozyme loci were used to assess differentiation between 11 regions located across the species range (Grant et al. 1987).
Grant et al. (1987) found large differences between Asian and North American populations, a division which has been confirmed in subsequent studies (Cunningham et al. 2009; Canino et al. 2010; Kim et al. 2010) and is also typical for other pelagic species in the northern Pacific (Walleye Pollock, O'Reilly et al. 2004).
This large difference between Asian and North American populations is indicative of recolonization of the Bering Sea following the last ice age when populations expanded north from glacial refugia (Canino et al. 2010).

In a more recent study of \fishName{} using microsatellite markers, Cunningham et al. (2009) calculated an average FST of 0.005 ($\pm$0.002) and found evidence for significant isolation by distance (IBD) along the coast of North America.
Thus, populations may be continuous, with no strong genetic distinction between adjacent samples; however, accumulated differences could result in samples taken from distant regions being quite different.
This type of pattern represents a particular challenge for management, as distant populations may still warrant independent management, but there is no clear region in which to draw management boundaries (Cunningham et al. 2009).
Despite the strong signature of IBD, Cunningham et al. (2009) also saw some modest support for barriers to gene flow between samples from the Strait of Georgia and the Washington coast, between Hecate Strait and the Strait of Georgia, and between these more southerly regions and Alaskan samples; they also detected the Asia/North American division observed in earlier studies.
Canino et al. (2010) extended the analysis of Cunningham et al. (2009) with the addition of a mtDNA locus (the NADPH subunit 2 gene); findings were similar with this new marker.
Due to the differentiation between Puget Sound and the Strait of Georgia, both Cunningham et al. (2009) and Canino et al. (2010) suggested that isolated populations may exist within fjords along the US and Canadian Pacific coast. Genetic differentiation between fjord and coastal populations has also been observed in Atlantic Cod (Knutsen et al. 2011) and Pacific Herring (Wildes et al. 2011).

IBD along the coast of North America was also seen by Liu et al. 2010 using sequence data from the control region of mtDNA.
However, genetic distance was not correlated with geographic distance across all populations so environmental factors, such as currents that affect larval dispersal, may also impact isolation between regions  (Liu et al. 2010).
Indeed, there was no significant difference between samples taken from Hecate Strait and from the Strait of Georgia and the division between Southeast Alaska and the BC coast was also not observed when assessed with the mtDNA control region (Liu et al. 2010).
Because mtDNA is clonally inherited without recombination, the allele sample size is only half that of nuclear genes (i.e., only one chromosome per individual rather than two) and all loci are linked, meaning mtDNA is treated as one locus.
In addition, \fishName{} appear to have relatively low mitochondrial sequence diversity, most likely a result of bottlenecks or founder effects following post-glacial recolonization (Liu et al. 2010).
Thus, these genetic markers may have lower power for detecting divergence than do microsatellites with their higher mutation rate and greater variability (Waples 1998).
In contrast, Canino et al. (2010) did find evidence of population structuring using two different mtDNA loci.
The loci used by Canino et al. (2010) had greater nucleotide diversity than did the locus used by Liu et al. (2010), and thus greater statistical power for detecting between population differences, which may explain the different results of these two studies.

Strognov et al. (2009a; 2009b; 2010), using a different suite of microsatellite markers from those used in the study by Cunningham et al. (2009), found no difference between samples taken off the west coast of Vancouver Island, from the Bering Sea, or from the Sea of Okhotsk although each of these regions differed from samples taken near the Kuril Islands.
However, these studies relied on only four microsatellites and it is quite likely that these markers did not have sufficient power to distinguish between populations.
Furthermore, the number of fish sampled within regions varied from 50 (from the Sea of Okhotsk) to 450 (from the Bering Sea); unbalanced sampling may result in a lack of power to detect genetic differentiation (Ryman et al. 2006).

Two studies have also examined fine scale structure in Asian \fishName{} populations.
Kim et al. (2010) segregated Korean populations into two groups, southern/western and eastern populations, using microsatellites.
While, Gwak and Nakayama (2011) identified three Korean populations, western, southern, and eastern, using a different suite of microsatellite markers and one mitochondrial locus.

To summarize, several studies, using a variety of molecular markers, have identified a distinction between North American and Asian \fishName{} populations and between Alaskan populations and those south of Dixon Entrance.
There is some weak evidence from the studies of Strognov et al. suggesting a lack of distinction across the Pacific or between Vancouver Island and Alaska.
However, given the low number of molecular markers used in those studies and the more robust findings from other studies, the suggestion of no structure should be discounted.
There is some evidence that samples taken off the coast of Washington and the west coast of Vancouver Island may be distinct from fish sampled within the Strait of Georgia or Puget Sound.
The hypothesis has also been raised that isolated populations may exist within fjords along the coast of British Columbia.
Evidence from Korean studies suggests that population differentiation can exist over relatively short distances.
To date little work has been done to determine if population structure exists within BC waters.
Given the findings of IBD along the coast and the suggestion that population structure can evolve over relatively short distances, additional work pertaining to population structure in BC would be of interest.

\appsection{RESEARCH RECOMMENDATIONS}

\subsection{SAMPLING DESIGN}

In order to address question one, genetic samples should be collected from spawning aggregations if possible.
In Hecate Strait, spawning has been observed around Horseshoe Island, Reef Island and Dogfish Bank's fishing grounds, and Rose Spit and Reef Island (Sinclair and Starr 2005).
Spawning sites have also been identified off the west coast of Vancouver Island and in the Strait of Georgia (Ketchen 1961).
No spawning sites have been identified in Queen Charlotte Sound (Sinclair and Starr 2005) but sampling efforts from February to March may capture spawning fish if they exist in this region.
Additional samples of \fishName{} should be obtained from the Gulf of Alaska, Puget Sound and the Washington coast; these samples could be obtained from researchers at the NOAA Fisheries Alaska Center who have already collected samples for previous studies.
It should be noted that the genotyping results from future work on Canadian populations cannot be compared directly with previous studies as microsatellites are notoriously difficult to standardize between laboratories; thus, samples from previously studied populations should be re-genotyped and analyzed together with newly collected samples.
In order to address question two, genetic sample should be collected in QCS and HS from non-spawning populations.
Both sets of samples should include adult fish rather than juveniles to minimize bias due to non-random sampling of families (Waples 1998).

Samples could consist of either fin clips or opercle punches from a minimum of 100 fish per region (WCVI, SoG, QCS and HS) both during and outside of the spawning season. Only a small amount of tissue (approximately 1 mm3) is needed.
Tissue should be stored in 95\% ethanol prior to use.
Because marine fishes tend to have low levels of genetic differentiation samples sizes must be larger than for anadromous or freshwater species.
Samples sizes of 100 individuals per regions are recommended here as statistical modeling indicates this sample size is sufficient to identify relatively low levels of divergence and that increased sample sizes beyond 100 individuals result in low marginal returns in increased power at the FST values seen in previous studies (i.e., FST = 0.005) (Waples 1998).

It is also possible to obtain DNA from scale or otolith samples (Nielsen and Hansen 2008).
Thus, it may be possible to use samples which have already been collected for aging or other uses provided sampling location, date and, ideally, storage conditions of any samples are known.
Sample storage is important for preservation of DNA for future genetic work.
Samples stored in poor conditions (i.e., high humidity, exposure to heat and/or sunlight) are more subject to DNA degradation (Nielsen and Hansen 2008).
While it may be possible to extract sufficient DNA for microsatellite analysis, the quality of the DNA should be checked prior to any attempts to amplify molecular markers; this can be done by measure the optical density of extracted DNA at 260 and 280 nm or by electrophoresis (Nielsen and Hansen 2008).
Contamination may also arise if samples are stored in batches.
Different washing techniques may be able to eliminate potential contamination (Mitchell et al. 2008).

\subsection{MOLECULAR MARKERS}

Twelve markers are suggested for the assessment of population structure within the area of interest (Table 1).
These markers were used in a previous study examining stock structure along the Pacific coast of North America (Cunningham et al. 2009).
If any of these markers is uninformative in the samples in question (e.g., are found to be monomorphic, are under selection, have large allele dropout or have null alleles; please see Glossary) additional microsatellite loci have also been used with some success in \fishName{} (Table 2) and could be tested in fish used here.
The facilities exist at the Pacific Biological Station, Molecular Genetics lab, for extracting DNA and genotyping fish using microsatellites.

In general, increasing the number of independent alleles (the number of alleles at a locus minus one summed across loci), either through increased numbers of loci or use of highly polymorphic loci, increases the precision of measurements of population differentiation (Kalinowski 2002).
The markers suggested here have been shown to have modest to high polymorphism in some populations of \fishName{}.

\subsection{STATISTICAL ANALYSIS}

Microsatellite markers should be checked for adherence to Hardy-Weinberg equilibrium (HWE) values.
Departures from HWE may be indicative of technical issues such as the presence of null alleles or large allele drop out (see Glossary), both of which result in a greater number of homozygotes than expected.
Alternatively, departures from HWE may indicate that loci are under selection or that multiple populations are being sampled.
The program MICROCHECKER can be used to calculate HWE and to distinguish between possible reasons for any deviations (van Oosterhout et al. 2004).

Neutrality (i.e., loci are not under selection) is a key assumption of most analyses of population structure.
Therefore, microsatellites should be checked for neutrality prior to subsequent analyses.
Selection can be tested with the method of Beaumont and Nichols (1996).
The program LOSITAN provides a convenient graphic user interface for this method (Antao et al. 2008).
Briefly, a neutral distribution of FST and heterozygosity values is generated using simulations of populations with the same sample size and average global FST as the actual samples.
Then each marker is plotted against this neutral distribution based on pairwise FST's and observed heterozygosity.
Loci that fall outside the 95\% credibility intervals of the neutral distribution are presumed to be under selection (Beaumont and Nichols 1996). 

Gene diversity in populations is typically characterized by allele frequencies, heterozygosity, and allelic richness, these quantities can be calculated using either Genepop (Rousset 2009; http://genepop.curtin.edu.au/) or FSTAT (Goudet 1995; www2.unil.ch/popgen/softwares/fstat.htm).
Both these programs can also be used to estimate pairwise FST; permutation tests conducted in FSTAT can determine if FST values are significant. 

The FST test described above pre-supposes that populations have been correctly identified, thus making pairwise FST's a good indicator of divergence between groups.
However, it is possible that the currently defined management boundaries, upon which sampling is based, do not accurately represent genetic boundaries.
The program STRUCTURE can be used to infer the number of genetic populations within a dataset (Pritchard et al. 2000).
The program can also be used to assign individuals to previously determined populations.
Thus, this program can also be used to determine of samples taken from the fishery are assigned back to the region from which samples were taken or if those individuals most likely originated from one of the other populations identified in response to question one.

IBD can be assessed in study populations by use of a Mantel test implemented in Genepop using the ISOLDE program.
The Mantel test is a statistical evaluation of the correlation between two semi-matrices, in this case pairwise genetic distance (i.e., FST) and geographic distance; a significant correlation between the semi-matrices is indicative of IBD.
Geographic distance between populations can be calculated from the latitude/longitude of sampling stations. 

\appsection{REFERENCES}

\appsection{TABLES}

\appsection{GLOSSARY OF GENETIC TERMS}

\begin{description}
\item[Allozymes] 
A variant of an amino acid within an enzyme which arises due to different alleles at a single locus; variants are detected by changes in electrophoretic mobility of the protein through a gel.
Typically variants do not result in a change in function of the protein and thus are considered evolutionarily neutral.
\item[Effective population size, $N_e$]
The number of individuals in a randomly mating population, with all individuals having equal reproductive success that would have the same level of genetic variation and inbreeding as that seen in the study population.
The effective population size is typically smaller (and often much smaller) than the census population size.
\item[FST]
FST (aka the fixation index) is a measure the reduction in heterozygosity due to inbreeding within a subpopulation.
FST can also be thought of as the proportion of genetic variation contained within a subpopulation (S) compared to the total genetic variation (T) combined across all subpopulations.
The statistic can be calculated based either on whether randomly drawn alleles are identical by descent (i.e., the same allele due to inheritance from a common ancestor) or identical in state (i.e., the same allele as a function of how frequently that allele occurs in the population).
Because it is rare to know if an allele is identical by descent (a detailed pedigree is necessary), FST is typically estimated based on identity in state and a variety of methods have been devised to this effect.
The most common method used for estimating FST is that of Weir and Cockerham (1984) 
\begin{equation}
F_\mli{ST} = \frac{Q_2 - Q_3}{1 - Q_3}
\end{equation}
where
$Q_2$ is the frequency of identical pairs of genes (i.e., the same allele at a locus) between individuals within subpopulations, and
$Q_3$ is the frequency of identical pairs of genes between subpopulations.
Values for FST range from 0 (panmixia) to 1 (complete isolation).
\item[Isolation by distance, IBD]
Populations which are geographically farther apart tend to be more genetically distinct because they are less likely to exchange migrants.
IBD is often found along an environmental gradient, within river systems or along coastlines. 
\item[Large (or upper) allele dropout]
Sometimes large alleles fail to be amplified in a PCR reaction.
This often occurs because of the preferential amplification of smaller alleles due to competition for primers and nucleotides within the PCR reaction (Wattier et al. 1998).
Once even a small imbalance occurs, the amplification process will result in the final PCR product containing an exponentially greater number of shorter alleles.
Upper allele dropout has been observed to occur more frequently when there is a large size difference between alleles in a heterozygous individual (Wattier et al. 1998).
Upper allele dropout results in an excess of homozygous individuals, particularly of individuals homozygous for smaller alleles.
The problem can sometimes be corrected by using a greater starting amount of DNA or by varying PCR conditions (i.e., increasing the extension time).
Occasionally, the problem is simply due to searching within an incorrect size range for alleles during analysis in which case the problem can easily be corrected by increasing the size range.
\item[Locus]
A specific location in the genome.
A variant at a locus is an allele.
\item[Microsatellites]
A segment of non-coding DNA which consists of short, repeated segments (typically 2, 3 or 4 basepair repeats).
The number of repeated segments may vary considerably between individuals.
These markers are inherited in a Mendelian fashion.
Microsatellites are considered evolutionarily neutral because they are non-coding although in some cases they may be so closely associated with a functional gene under selection that they appear to be under selection.
The assumption of neutrality can be tested using the program LOSITAN (Antao et al. 2008) which assesses the level of genetic differentiation between populations (FST) compared to the heterozygosity of a locus
\item[Molecular markers]
Any fragment of DNA which can be reliably identified or located within the genome.
\item[Null allele]
An allele at a locus that contains a mutation in the priming site so that the allele is not amplified during PCR.
Null alleles are often assumed to be present if a population has an unexpectedly high number of individuals that are homozygous at that locus.
\item[Polymerase Chain Reaction, PCR]
PCR is a biochemical technique to quickly and reliable generate millions of copies of a segment of DNA.
With new refinements of the technique it is possible to generate new copies from as little as one copy of starting material.
The method relies on thermalcycling and the heat stable enzyme Taq polymerase to assemble DNA strands in a manner similar to that of a cell.
The PCR reaction mixture contains the purified DNA template; the nucleotides adenine, cytosine, guanine and thymine (collectively, dNTPs); Taq polymerase; a buffer solution and other stabilizers (typically $\text{MgCl}_2$); and sense and antisense primers, which can either be specific to a targeted segment or generic depending on the application.
DNA strands are heated to $96^\circ$C until they denature (i.e., the hydrogen bonds between basepairs are broken and the DNA becomes single stranded).
The temperature is lowered until the primers anneal to the single stranded DNA; the annealing temperature is usually, but not always, about $3^\circ$C below the melting point ($\text{T}_\text{m}$) of the primers.
The temperature is then raised to $72^\circ$C, the optimum temperature for the polymerase, and the new DNA strand is synthesized by adding dNTPs along the template strand starting from the end of the primer.
The cycle is repeated with the newly synthesized strands serving as a template in the next round; thus, resulting in an exponential increase in copies of target DNA.
PCR is most efficient when the primers match the target DNA exactly, but by manipulation of annealing temperature and reaction mix, it is possible to have some mismatches in the priming site.
This means that primers developed for one species can often be used to amplify loci in another, closely related species that may have only a few differences in the priming site for the locus in question.
\end{description}
